### ISSUE TITLE
Add a click target graphic

### ISSUE THREAD

**Issue by melanie** [SolidGoldStudios/SecretSymphony#25]
Once the NevMesh-ing is done, we should add in a neat-looking graphic or effect to indicate where you've clicked for Kay to move to.

**Comment by melanie** [comment]
I've added in some basic cursors, but they're far from perfect! Here are some issues I could use help with:

- [ ] The cursors appear underneath other UICanvas elements, like the dialog box, menu button, and main menu
- [ ] The cursor doesn't reset back to the blue arrow after you hover over an Interact- or Speak-tagged item
- [ ] Handling this in the MouseCursor script is probably not ideal; Dustin says they should be handled in the Interaction stuff we already have

Here's a Loom video of what I've just merged into master:
https://www.loom.com/share/1b6501109141462182b972a2d7cc8b4f
